utils/auth.py

```
@@ -0,0 +1,84 @@
+                detail="유효하지 않은 검증(이메일,유저아이디 없는 값 존재)",
+                headers={"WWW-Authenticate": "Bearer"},
+            )
+
+        return TokenData(email=email, user_id=user_id)
+
+    except JWTError:
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail="유효하지 않은 검증",
+            headers={"WWW-Authenticate": "Bearer"},
+        )
```

Ruff (0.14.11)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)